api/config/config.go

```diff
@@ -110,6 +111,8 @@ type ImageBuilderConfig struct {
 	BuildNamespace               string `envconfig:"IMG_BUILDER_NAMESPACE" default:"mlp"`
 	DockerRegistry               string `envconfig:"IMG_BUILDER_DOCKER_REGISTRY"`
 	BuildTimeout                 string `envconfig:"IMG_BUILDER_TIMEOUT" default:"10m"`
+	// How long to keep the image building job resource in the Kubernetes cluster. Default: A week (168 hours).
+	Retention time.Duration `envconfig:"IMG_BUILDER_RETENTION" default:"168h"`
```

Comment:
I think we should reduce the default retention to 2 days. Having 1 week as default is not useful because the docker container will already be deleted by kubelet and user won't be able to fetch the logs anyway.